services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/email/EmailSenderService.kt

```diff
@@ -127,8 +127,7 @@ class EmailSenderService(
         mailSender.send(mimeMessage)
     }
 
-    private fun defineInviterInfo(name: String?, email: String): String =
-        if (name.isNullOrBlank()) "Asioija $email" else "$name ($email)"
+    private fun inviterInfo(name: String, email: String): String = "$name ($email)"
```

Comment:
Name and email could be just split into two separate parameters now that there's no branching.

Reply:
Fixed.